### ISSUE TITLE
[SearchBundle] Populate command failing

### ISSUE THREAD

**Issue by wesleylancel** [Kunstmaan/KunstmaanBundlesCMS#1704]
After some recent changes in the SearchBundle I'm running in to some issues, first when executing `kuma:search:populate full`:

```
[Symfony\Component\Console\Exception\LogicException]
  Cannot retrieve helper "question" because there is no HelperSet defined. Did you forget to add your command to the application or to set the application on the command using the setApplication() method? You can also set the HelperSet directly using the setHelperSet() method.
```

When I run the setup command itsself, it will ask about unavailable language analyzers, however I did configure those:

```
kunstmaan_search:
    analyzer_languages:
        nl_BE: { analyzer: 'dutch' }
        nl_NL: { analyzer: 'dutch' }
        fr_BE: { analyzer: 'french' }
```

**Comment by cv65kr** [comment]
@wesleylancel, #1635 i will check this bug tomorrow

**Comment by Numkil** [comment]
@wesleylancel the crash has been fixed in above PR. Also at the time of writing this the languages in the analyzer_languages have to be defined in lower case or else it will not work. I will make an other PR that makes sure you don't have to worry about the lower/upper case in that config.

**Comment by Numkil** [comment]
@wesleylancel Since both related PR's have been merged in 5.0 I will be closing this ticket. If you have any other remarks do not hesitate to reopen this ticket.

**Comment by wesleylancel** [comment]
Thanks guys!
